docs/sdk.md

```diff
@@ -203,3 +203,38 @@ and all the semantics of the Ethereum state machine.
 
 Within `x/eth`, transactions sent to particular addresses can be handled in unique ways,
 for instance to handle IBC and staking.
+
+# cosmos-sdk-cli 
```

Comment:
This tooling is WIP and not ready for usage. Its currently way out of sync with the codebase and needs updating. I think we should leave out mentions to it if we can.

Reply:
yes, zach is only working on reorg, and I will work on content on another PR (fixing links, removing/updating depracted stuff, etc.